src/components/StatusIndicator/__tests__/__snapshots__/StatusIndicator.spec.js.snap

```diff
@@ -43,40 +45,22 @@ exports[`StatusIndicator renders 1`] = `
       status="incomplete"
     />
     <Button
-      className="security--status-indicator__retry-button"
+      className="security--status-indicator__button--retry"
```

Comment:
Hey I noticed this class isn't the same as before 😅 

Previously it was `security--status-indicator__retry-button`
How do you feel about changing it back, just in case someone's tests or styles are relying on it?